docs/reference/webhook.mdx

```diff
@@ -84,5 +88,90 @@ The webhook sends four types of experiment update events. If you only want to ta
 }
 ```
 
+## Flagging Webhooks
+
+The flagging webhook is a good solution when a custom application needs to display 
+or take action on the most up to date state of a flag.
+
+### Flag Configuration Updated
+
+The flag configuration updated webhook is sent when a flag or its associated models, such as assignments, targeting rules, environments or audiences, are updated.
```

Comment:
"The Flag Configuration Updated Webhook" capitalize to be consistent with "The Experiment Update Webhook" on line 7?